_includes/components/search.html

```diff
@@ -0,0 +1,11 @@
+<form accept-charset="UTF-8" action="https://search.usa.gov/search" class="usa-search usa-search-small js-search-form" method="get">
```

Comment:
Nitpicky, but the `id` attribute on the form didn't make the transition, its possible that downstream users maybe using it, is there a downside to restoring it? Are there multiple "search form"s in the markup?

Reply:
Even if there are, they are at the top of the page, so I doubt anyone is using them to jump.

Reply:
Maybe not, but I just used it to target the form to temporarily address the invisible search input issue in Federalist :) so its feasible (though I agree unlikely) another user might use it for something.

Unless its causing a problem and/or you feel strongly about it,  I would prefer to leave it just in case someone IS using it, so this small upgrade doesn't break their site.

Reply:
Approving.